src/utils/helpers.coffee

```diff
@@ -64,6 +65,15 @@ exports.excludeFile = (fileName, options) ->
             # Only instrument files that are inside the project.
             excluded = true
 
+        # For each exclude value try to use it as a pattern to exclude files
+        exclude.map (pattern) ->
+            glob.sync pattern,
```

Comment:
No, never mind, yours is about twice as fast as mine in a quick test I just tried.  -_-

This is because of the way I'm handling folders.  Your version keeps the old prefix matches, so excludes like "/node_modules" still match everything that starts with "/node_modules", whereas mine needs to look through every glob found (which could be a lot, if someone specified /node_modules/**/*) looking for a parent folder with _.startsWith.  :P